Sources/PrivoSDK/model/OAuthToken.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+struct OAuthToken: Codable {
+    var grant_type: String = "client_credentials"
+    var scope: String = "openid profile user_profile update_password_link TRUST"
+    let client_id: String
+    let client_secret: String
+}
+
+struct TokenResponse: Decodable {
+    let access_token: String
+    let token_type: String
+    let expires_in: Int
+    let scope: String
+}
+
+extension OAuthToken {
+    func toQueryItems() -> [URLQueryItem] {
+        let mirror = Mirror(reflecting: self)
```

Comment:
OAuth token do not related to URL or to QueryItems
it means that this converter should be in another place

Reply:
after switching request to gw token this code removed as unused